{"abstract":[{"text":"Handle the revocation of a credential.","type":"text"}],"kind":"symbol","schemaVersion":{"major":0,"minor":3,"patch":0},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"handleCredentialRevocation"},{"kind":"text","text":"("},{"kind":"externalParam","text":"issuer"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":", "},{"kind":"externalParam","text":"message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?, "},{"kind":"externalParam","text":"challenge"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:6DIDSDK9ChallengeC","text":"Challenge"},{"kind":"text","text":"?, "},{"kind":"externalParam","text":"claimReference"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:6DIDSDK14ClaimReferenceC","text":"ClaimReference"},{"kind":"text","text":", "},{"kind":"externalParam","text":"errors"},{"kind":"text","text":": ["},{"kind":"typeIdentifier","preciseIdentifier":"s:13PingOneWallet0C9ExceptionO","text":"WalletException","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException"},{"kind":"text","text":"]) -> "},{"kind":"typeIdentifier","preciseIdentifier":"s:Sb","text":"Bool"}]}],"kind":"declarations"},{"parameters":[{"name":"issuer","content":[{"inlineContent":[{"type":"text","text":"ApplicationInstanceID of the credential issuer"}],"type":"paragraph"}]},{"name":"message","content":[{"inlineContent":[{"type":"text","text":"Optional string message"}],"type":"paragraph"}]},{"name":"challenge","content":[{"inlineContent":[{"type":"text","text":"Optional challenge"}],"type":"paragraph"}]},{"name":"claimReference","content":[{"inlineContent":[{"type":"text","text":"ClaimReference for the revoked credential"}],"type":"paragraph"}]},{"name":"errors","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"List of any errors that occurred while revoking the credential"}]}]}],"kind":"parameters"},{"kind":"content","content":[{"type":"heading","text":"Return Value","anchor":"return-value","level":2},{"inlineContent":[{"text":"True if the user has accepted the credential revocation, False if the user has rejected the credential revocation","type":"text"}],"type":"paragraph"}]}],"variants":[{"paths":["\/documentation\/pingonewallet\/walletcallbackhandler\/handlecredentialrevocation(issuer:message:challenge:claimreference:errors:)"],"traits":[{"interfaceLanguage":"swift"}]}],"hierarchy":{"paths":[["doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet","doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletCallbackHandler"]]},"identifier":{"url":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletCallbackHandler\/handleCredentialRevocation(issuer:message:challenge:claimReference:errors:)","interfaceLanguage":"swift"},"metadata":{"modules":[{"name":"PingOneWallet"}],"title":"handleCredentialRevocation(issuer:message:challenge:claimReference:errors:)","externalID":"s:13PingOneWallet0C15CallbackHandlerP26handleCredentialRevocation6issuer7message9challenge14claimReference6errorsSbSS_SSSg6DIDSDK9ChallengeCSgAK05ClaimM0CSayAA0C9ExceptionOGtF","fragments":[{"text":"func","kind":"keyword"},{"text":" ","kind":"text"},{"text":"handleCredentialRevocation","kind":"identifier"},{"text":"(","kind":"text"},{"text":"issuer","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":", ","kind":"text"},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?, ","kind":"text"},{"text":"challenge","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Challenge","kind":"typeIdentifier","preciseIdentifier":"s:6DIDSDK9ChallengeC"},{"text":"?, ","kind":"text"},{"text":"claimReference","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"ClaimReference","kind":"typeIdentifier","preciseIdentifier":"s:6DIDSDK14ClaimReferenceC"},{"text":", ","kind":"text"},{"text":"errors","kind":"externalParam"},{"text":": [","kind":"text"},{"text":"WalletException","kind":"typeIdentifier","preciseIdentifier":"s:13PingOneWallet0C9ExceptionO"},{"text":"]) -> ","kind":"text"},{"text":"Bool","kind":"typeIdentifier","preciseIdentifier":"s:Sb"}],"role":"symbol","symbolKind":"method","roleHeading":"Instance Method","required":true},"sections":[],"references":{"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletCallbackHandler/handleCredentialRevocation(issuer:message:challenge:claimReference:errors:)":{"kind":"symbol","url":"\/documentation\/pingonewallet\/walletcallbackhandler\/handlecredentialrevocation(issuer:message:challenge:claimreference:errors:)","title":"handleCredentialRevocation(issuer:message:challenge:claimReference:errors:)","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletCallbackHandler\/handleCredentialRevocation(issuer:message:challenge:claimReference:errors:)","type":"topic","required":true,"role":"symbol","abstract":[{"text":"Handle the revocation of a credential.","type":"text"}],"fragments":[{"text":"func","kind":"keyword"},{"text":" ","kind":"text"},{"text":"handleCredentialRevocation","kind":"identifier"},{"text":"(","kind":"text"},{"text":"issuer","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":", ","kind":"text"},{"text":"message","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?, ","kind":"text"},{"text":"challenge","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Challenge","kind":"typeIdentifier","preciseIdentifier":"s:6DIDSDK9ChallengeC"},{"text":"?, ","kind":"text"},{"text":"claimReference","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"ClaimReference","kind":"typeIdentifier","preciseIdentifier":"s:6DIDSDK14ClaimReferenceC"},{"text":", ","kind":"text"},{"text":"errors","kind":"externalParam"},{"text":": [","kind":"text"},{"text":"WalletException","kind":"typeIdentifier","preciseIdentifier":"s:13PingOneWallet0C9ExceptionO"},{"text":"]) -> ","kind":"text"},{"text":"Bool","kind":"typeIdentifier","preciseIdentifier":"s:Sb"}]},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletCallbackHandler":{"title":"WalletCallbackHandler","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"WalletCallbackHandler"}],"abstract":[],"role":"symbol","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletCallbackHandler","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"WalletCallbackHandler"}],"url":"\/documentation\/pingonewallet\/walletcallbackhandler","type":"topic"},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet":{"title":"PingOneWallet","url":"\/documentation\/pingonewallet","identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet","kind":"symbol","role":"collection","type":"topic","abstract":[{"type":"text","text":"This defines the usage and interfaces of the PingOne Wallet SDK for iOS to interact with PingOne credentialing services. The SDK provides for a user to receive, save, and share credentials for identification and verification purposes."}]},"doc://com.pingidentity.PingOneWallet/documentation/PingOneWallet/WalletException":{"kind":"symbol","role":"symbol","navigatorTitle":[{"text":"WalletException","kind":"identifier"}],"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"WalletException","kind":"identifier"}],"identifier":"doc:\/\/com.pingidentity.PingOneWallet\/documentation\/PingOneWallet\/WalletException","abstract":[],"title":"WalletException","url":"\/documentation\/pingonewallet\/walletexception"}}}